zebra-chain/src/orchard/keys.rs

```diff
@@ -982,7 +982,7 @@ impl PartialEq<[u8; 32]> for TransmissionKey {
 /// An _outgoing cipher key_ for Orchard note encryption/decryption.
 ///
 /// <https://zips.z.cash/protocol/nu5.pdf#saplingandorchardencrypt>
-// TODO: derive `OutgoingCipherKey`: https://github.com/ZcashFoundation/zebra/issues/2041
+// TODO: derive `OutgoingCipherKey`: <https://github.com/ZcashFoundation/zebra/issues/2041>
```

Comment:
Optional: Not needed.
```suggestion
// TODO: derive `OutgoingCipherKey`: https://github.com/ZcashFoundation/zebra/issues/2041
```

Reply:
Resolved in [36fc845](https://github.com/ZcashFoundation/zebra/pull/4494/commits/36fc8453d4239291deb6e3f692a0d6b9253f17e8).